docs/status.md

```diff
@@ -52,94 +52,94 @@ one of the following tracking labels.
 | or                       |        yes         |        yes*        |        yes*         |         yes          |        no        |
-| outfeed                  |         no         |        yes*        |         no          |          no          |        no        |
+| outfeed                  |         no         | yes(need-revisit)  |         no          |          no          |        no        |
 | pad                      |         no         |        yes*        |        yes*         |          no          |        no        |
 | popcnt                   |         no         |        yes*        |        yes*         |         yes          |        no        |
 | power                    |         no         |        yes*        |        yes*         |         yes          |        no        |
-| real_dynamic_slice       |         no         |        yes*        |         no          |         yes          |        no        |
+| real_dynamic_slice       |         no         | yes(need-revisit)  |         no          |         yes          |        no        |
 | real                     |         no         |        yes*        |        yes*         |         yes          |        no        |
-| recv                     |         no         |        yes*        |         no          |          no          |        no        |
+| recv                     |         no         | yes(need-revisit)  |         no          |          no          |        no        |
 | reduce                   |         no         |        yes*        |        yes*         |  yes(need-revisit)   |        no        |
 | reduce_precision         |         no         |        yes*        |        yes*         |         yes          |        no        |
-| reduce_scatter           |         no         |        yes*        |         no          |          no          |        no        |
+| reduce_scatter           |         no         | yes(need-revisit)  |         no          |          no          |        no        |
 | reduce_window            |         no         |        yes*        |        yes*         |          no          |        no        |
 | remainder                |        yes         |        yes*        |        yes*         |         yes          |        no        |
-| replica_id               |         no         |        yes*        |  yes(need-revisit)  |         yes          |        no        |
-| reshape                  |        yes         |        yes         |     infeasible      |         yes          |       yes        |
-| return                   |         no         |        yes*        |         no          |         yes          |        no        |
+| replica_id               |         no         | yes(need-revisit)  |  yes(need-revisit)  |         yes          |        no        |
+| reshape                  |        yes         | yes(need-revisit)  |     infeasible      |         yes          |       yes        |
```

Comment:
I don't think that we need to revisit `reshape`. #170 has already determined that it's a "yes".

Reply:
Thanks! in the new version updated today, it respects all the recent changes to the repo.